### ISSUE TITLE
MD024 (Multiple headers with same content) should only trigger under the same nesting

### ISSUE THREAD

**Issue by H-G-Hristov** [DavidAnson/vscode-markdownlint#238]
I took this from: https://github.com/markdownlint/markdownlint/issues/175

In my opinion

```
# Change log

## 1.0.0

### Bug fixes
...

### Features
...

### Bug fixes
...
```

should trigger an offense.

But

```
# Change log

## 1.0.0

### Bug fixes
...

### Features
...

## 0.0.1

### Bug fixes
...
```

should not.

**Comment by DavidAnson** [comment]
This is what the `siblings_only` parameter is for. There is an example of your scenario in the documentation for this rule: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md024
